docs/sources/userguide/dockerlinks.md

```diff
@@ -331,6 +333,74 @@ allowing linked communication to continue.
     . . .
     172.17.0.9  db
 
+## Connect with Networks
+
+> Note: This feature is experimental
+
+Docker's linking system hardcodes a link between two running containers. In order to 
+change a link, you must restart a container. To avoid this, the 
+[Ambassador Pattern](../articles/ambassador_pattern_linking) was developed which requires
+ deploying additional containers to handle linking. To simplify this, we have introduced
+Networks.
+
+### What are Networks?
+
+Networks are simply a group of containers that can communicate with eachother.
+They are useful for segmenting groups of containers and applying some form of policy, via
+`iptables` for example, or to organise all containers for a given application stack.
+
+### Example
+
+Lets create a network for our application called `yoda`:
+
+    $ docker network create yoda
+    a10ef560538f6221c225957c9dc3d497c749e34c3f92c2964385601ad02bca53
+
+The `yoda` application consists of one nginx container and one database. Let's create those now,
+but lets assign them static IP addressses.
+
+    $ docker run -itd --name web1 --net=yoda -l ip=192.168.99.10 nginx
+    dc50c677fbb7acdcd150c7f9109958e50398a62a62c500189ce723ac498c618e
+    $ docker run -itd --name db1 --net=yoda -l ip=192.168.88.20 redis
+    153a96bbae2e7cf48594ffd84691328e62e19f8ac2ec53ad492f1518b918a4c0
+
+Ok, now lets see if they can communicate!
+
+    $ docker exec web ping 192.168.99.20
```

Comment:
minor nit: your names are **web1** and **db1** above, but the ping examples use **web** and **db**